pkg/controller/monitoring.go

```diff
@@ -168,6 +168,18 @@ func (c *Controller) updateHealthStatusForTenant(tenant *miniov2.Tenant) error {
 		return nil
 	}
 
+	// Add back "Usable Capacity" & "Internal" values in Tenant Status and in the UI
+	// How much is available: "Usable Capacity" in UI comes from "tenant.Status.Usage.Capacity"
+	// How much is used: "Internal" in UI comes from "tenant.Status.Usage.Usage"
+	UsedSpace := int64(0)      // How much is used
```

Comment:
@cniackz do you know how often do we update the health status of this tenant ?

Reply:
Hello @vadmeste 

Very good question of yours!.
Ok, I am not sure if this is the same on every cluster but in my testing cluster it is around every 3 minutes or so, look:
```
I0924 09:31:04.963121   12351 controller.go:71] Starting MinIO Operator
I0924 09:31:04.966878   12351 controller.go:130] Could not determine HOSTNAME, defaulting to pod name: operator-pod
I0924 09:31:04.986608   12351 main-controller.go:278] Setting up event handlers
I0924 09:31:05.003808   12351 main-controller.go:502] Using Kubernetes CSR Version: v1
I0924 09:31:05.003825   12351 main-controller.go:522] STS Api server is not enabled, not starting
I0924 09:31:05.003876   12351 leaderelection.go:250] attempting to acquire leader lease minio-operator/minio-operator-lock...
I0924 09:31:05.008700   12351 leaderelection.go:260] successfully acquired lease minio-operator/minio-operator-lock
I0924 09:31:05.008730   12351 main-controller.go:551] operator-pod: I am the leader, applying leader labels on myself
I0924 09:31:05.008790   12351 main-controller.go:397] Waiting for Upgrade Server to start
I0924 09:31:05.008799   12351 main-controller.go:401] Starting Tenant controller
I0924 09:31:05.008801   12351 main-controller.go:404] Waiting for informer caches to sync
I0924 09:31:05.008805   12351 main-controller.go:409] Starting workers
I0924 09:31:05.008811   12351 main-controller.go:442] Console TLS is not enabled
I0924 09:31:05.008868   12351 main-controller.go:353] Starting HTTP Upgrade Tenant Image server
E0924 09:31:05.011300   12351 main-controller.go:565] failed to patch operator leader pod: pods "operator-pod" not found
I0924 09:31:10.061756   12351 status.go:89] Hit conflict issue, getting latest version of tenant
I0924 09:31:10.061876   12351 status.go:89] Hit conflict issue, getting latest version of tenant
I0924 09:31:10.078741   12351 monitoring.go:183] Hello Cesar it is me the code you are trying to measure how often is triggered!.
I0924 09:31:10.079092   12351 monitoring.go:183] Hello Cesar it is me the code you are trying to measure how often is triggered!.
I0924 09:31:10.089827   12351 pdb.go:180] PodDisruptionBudget: v1
I0924 09:31:10.210005   12351 status.go:89] Hit conflict issue, getting latest version of tenant
I0924 09:31:11.017647   12351 monitoring.go:183] Hello Cesar it is me the code you are trying to measure how often is triggered!.
I0924 09:34:15.170811   12351 status.go:89] Hit conflict issue, getting latest version of tenant
I0924 09:34:15.191228   12351 monitoring.go:183] Hello Cesar it is me the code you are trying to measure how often is triggered!.
I0924 09:37:15.132691   12351 status.go:89] Hit conflict issue, getting latest version of tenant
I0924 09:37:15.144764   12351 monitoring.go:183] Hello Cesar it is me the code you are trying to measure how often is triggered!.
I0924 09:40:15.156791   12351 status.go:89] Hit conflict issue, getting latest version of tenant
I0924 09:40:15.175026   12351 monitoring.go:183] Hello Cesar it is me the code you are trying to measure how often is triggered!.
I0924 09:43:15.158742   12351 status.go:89] Hit conflict issue, getting latest version of tenant
I0924 09:43:15.177956   12351 monitoring.go:183] Hello Cesar it is me the code you are trying to measure how often is triggered!.
I0924 09:46:15.174892   12351 monitoring.go:183] Hello Cesar it is me the code you are trying to measure how often is triggered!.
I0924 09:49:15.168624   12351 status.go:89] Hit conflict issue, getting latest version of tenant
I0924 09:49:15.189140   12351 monitoring.go:183] Hello Cesar it is me the code you are trying to measure how often is triggered!.
I0924 09:52:15.167943   12351 status.go:89] Hit conflict issue, getting latest version of tenant
I0924 09:52:15.179993   12351 monitoring.go:183] Hello Cesar it is me the code you are trying to measure how often is triggered!.
I0924 09:55:15.170587   12351 status.go:89] Hit conflict issue, getting latest version of tenant
I0924 09:55:15.188218   12351 monitoring.go:183] Hello Cesar it is me the code you are trying to measure how often is triggered!.
I0924 09:58:15.171436   12351 status.go:89] Hit conflict issue, getting latest version of tenant
I0924 09:58:15.185906   12351 monitoring.go:183] Hello Cesar it is me the code you are trying to measure how often is triggered!.
I0924 10:01:15.162957   12351 status.go:89] Hit conflict issue, getting latest version of tenant
I0924 10:01:15.184257   12351 monitoring.go:183] Hello Cesar it is me the code you are trying to measure how often is triggered!.
I0924 10:04:15.164417   12351 monitoring.go:183] Hello Cesar it is me the code you are trying to measure how often is triggered!.
I0924 10:07:15.163187   12351 status.go:89] Hit conflict issue, getting latest version of tenant
I0924 10:07:15.178973   12351 monitoring.go:183] Hello Cesar it is me the code you are trying to measure how often is triggered!.
I0924 10:10:15.198434   12351 monitoring.go:183] Hello Cesar it is me the code you are trying to measure how often is triggered!.
I0924 10:13:15.165845   12351 status.go:89] Hit conflict issue, getting latest version of tenant
I0924 10:13:15.185202   12351 monitoring.go:183] Hello Cesar it is me the code you are trying to measure how often is triggered!.
I0924 10:16:15.176782   12351 status.go:89] Hit conflict issue, getting latest version of tenant
I0924 10:16:15.195578   12351 monitoring.go:183] Hello Cesar it is me the code you are trying to measure how often is triggered!.
I0924 10:19:15.146610   12351 status.go:89] Hit conflict issue, getting latest version of tenant
I0924 10:19:15.156557   12351 monitoring.go:183] Hello Cesar it is me the code you are trying to measure how often is triggered!.
I0924 10:22:15.153626   12351 status.go:89] Hit conflict issue, getting latest version of tenant
I0924 10:22:15.163237   12351 monitoring.go:183] Hello Cesar it is me the code you are trying to measure how often is triggered!.
I0924 10:25:15.167131   12351 status.go:89] Hit conflict issue, getting latest version of tenant
I0924 10:25:15.183370   12351 monitoring.go:183] Hello Cesar it is me the code you are trying to measure how often is triggered!.
I0924 10:28:15.197661   12351 monitoring.go:183] Hello Cesar it is me the code you are trying to measure how often is triggered!.
I0924 10:31:15.183035   12351 status.go:89] Hit conflict issue, getting latest version of tenant
I0924 10:31:15.201111   12351 monitoring.go:183] Hello Cesar it is me the code you are trying to measure how often is triggered!.
I0924 10:34:15.194281   12351 status.go:89] Hit conflict issue, getting latest version of tenant
I0924 10:34:15.215375   12351 monitoring.go:183] Hello Cesar it is me the code you are trying to measure how often is triggered!.
I0924 10:37:15.195144   12351 status.go:89] Hit conflict issue, getting latest version of tenant
I0924 10:37:15.211761   12351 monitoring.go:183] Hello Cesar it is me the code you are trying to measure how often is triggered!.
I0924 10:40:15.176107   12351 status.go:89] Hit conflict issue, getting latest version of tenant
I0924 10:40:15.193263   12351 monitoring.go:183] Hello Cesar it is me the code you are trying to measure how often is triggered!.
I0924 10:43:15.183714   12351 status.go:89] Hit conflict issue, getting latest version of tenant
I0924 10:43:15.201730   12351 monitoring.go:183] Hello Cesar it is me the code you are trying to measure how often is triggered!.
I0924 10:46:15.206889   12351 monitoring.go:183] Hello Cesar it is me the code you are trying to measure how often is triggered!.
I0924 10:49:15.194909   12351 status.go:89] Hit conflict issue, getting latest version of tenant
I0924 10:49:15.212799   12351 monitoring.go:183] Hello Cesar it is me the code you are trying to measure how often is triggered!.

```

Reply:
@cniackz I mean there is `Hit conflict issue, getting latest version of tenant` after your debug log so many times. I believe something are unreasonable here. Could you finger out it and fix it?

Reply:
Certainly, understood. We should address that issue by debugging and resolving it in a separate pull request as is not related to this change/addition.